src/screens/NewAnalytics/NewAnalyticsTypes.res

```diff
@@ -13,7 +13,7 @@ type dimension = [
   | #authentication_type
 ]
 type status = [#charged | #failure]
-type metrics = [#payment_processed_amount | #payment_success_rate]
+type metrics = [#payment_processed_amount | #payment_success_rate | #payment_distribution]
```

Comment:
its not a diff metric its uses `#payment_success_rate` metric itself, with a group by value

Reply:
okay